pkg/tsdb/cloudwatch/routes/regions_test.go

```diff
@@ -0,0 +1,99 @@
+package routes
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models/resources"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type RegionsService struct {
+	mock.Mock
+}
+
+func (r *RegionsService) GetRegions() ([]resources.ResourceResponse[resources.Region], error) {
+	args := r.Called()
+	mockResponse := args.Get(0)
+	if mockResponse == nil {
+		return nil, args.Error(1)
+	}
+	return mockResponse.([]resources.ResourceResponse[resources.Region]), args.Error(1)
+}
+
+func TestRegionsRoute(t *testing.T) {
+	reqCtxFunc := func(_ context.Context, pluginCtx backend.PluginContext, region string) (reqCtx models.RequestContext, err error) {
```

Comment:
This doesn't seem to be needed -- I think we're at a lower level here. You can replace it with `nil` in the lines below.
```suggestion
```